pkg/client/helper.go

```diff
@@ -245,8 +246,9 @@ func InClusterConfig() (*Config, error) {
 		Host:        "https://" + net.JoinHostPort(os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")),
 		Version:     "v1beta3",
 		BearerToken: string(token),
-		// TODO: package certs along with the token
-		Insecure: true,
+		TLSClientConfig: TLSClientConfig{
```

Comment:
Appears so. I'll do a stat check, and include it in the TLS config if it exists, and log if it doesn't. https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/apiserver/authn.go#L128